### PR TITLE
Fix flaky tests for quantize and requantize

### PR DESCRIPTION
**Pull request by xinyu-intel**
## Description ##
fix flaky tests for quantize and requantize in #12028 #11747 

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

@pengzhao-intel @marcoabreu

### CODE REVIEW

**Comment by xinyu-intel** [comment]
Tests have passed.
```
[INFO] 9995 of 10000: Setting test np/mx/python random seeds, use MXNET_TEST_SEED=2127644814 to reproduce.
[INFO] 9996 of 10000: Setting test np/mx/python random seeds, use MXNET_TEST_SEED=2127644814 to reproduce.
[INFO] 9997 of 10000: Setting test np/mx/python random seeds, use MXNET_TEST_SEED=2127644814 to reproduce.
[INFO] 9998 of 10000: Setting test np/mx/python random seeds, use MXNET_TEST_SEED=2127644814 to reproduce.
[INFO] 9999 of 10000: Setting test np/mx/python random seeds, use MXNET_TEST_SEED=2127644814 to reproduce.
[INFO] 10000 of 10000: Setting test np/mx/python random seeds, use MXNET_TEST_SEED=2127644814 to reproduce.
ok

----------------------------------------------------------------------
Ran 1 test in 62.163s

OK
INFO:root:Nosetests terminated with exit code 0
```

**Comment by marcoabreu** [comment]
Thanks a lot for addressing this so quickly!

I noticed that the testing seed is always the same. Would you mind un-setting it and using random ones?

**Comment by xinyu-intel** [comment]
@marcoabreu okay, still can pass. I paste wrong log:)

File: tests/python/quantization/test_quantization.py [hunk 1]
```diff
@@ -121,7 +120,7 @@ def check_requantize(shape, min_calib_range=None, max_calib_range=None):
                                                                           max_range.asscalar(),
                                                                           min_calib_range=min_calib_range,
                                                                           max_calib_range=max_calib_range)
-        assert_almost_equal(qdata_int8.asnumpy(), qdata_int8_np)
+        assert_almost_equal(qdata_int8.asnumpy(), qdata_int8_np, atol = 1)
```
Comment (haojin2) [inline tests/python/quantization/test_quantization.py]: Why do we allow `atol=1`?

**Comment by xinyu-intel** [comment]
@haojin2 https://github.com/apache/incubator-mxnet/issues/11747

**Comment by haojin2** [comment]
@xinyu-intel Thanks for the investigation.

**Comment by marcoabreu** [comment]
Sorry, I'm currently swamped with high priority tasks and I don't have time to review your pull request.

**Pull request merged**